docs/howto/stack_version_support.md

```diff
@@ -0,0 +1,80 @@
+Intro...
+
+TBD: When to reconsider the version constraint.
+
+
+Support for all versions since the release of the package
+----
+Example: `>=7.16.0`.
+
+With this approach, the released package can be used in any version since the
+release of the package.
+
+Pros:
+* No need to keep multiple development branches, no need for backports.
+
+Cons:
+* False premise, difficult to ensure compatibility with future majors.
+
+Recommendation: Not recommended.
+
+Support for multiple majors
+----
+Example: `^7.14.0 || ^8.0.0`.
+
+With this approach, the released package can be used in any tested major version
+since the release of the package.
+
+Pros:
+* No need to keep multiple development branches, no need for backports, same
+  package for 7.x and 8.x.
+* Less disruptive for users.
+
+Cons:
```

Comment:
* you may end up with a train: `^7.16.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0`

Reply:
Why would this be a con? If a package developer needs to maintain a package for all these versions, wouldn't it be better if they can do it with a single package instead of with many of them?

Reply:
It's easy to forget about extending the version range - new person, new hire, person unaware of such constraint. It's my personal opinion, but "exposing the package until revoked/withdrawn" seems to be natural to me. I can't achieve that with just ">= 7.16.0" constraint.

Reply:
Umm, so this is really a con for all cases except `>=`, right? In all the other cases you need to modify the constraints every time a new version needs to be supported.

Reply:
Yes, that's unfortunate. I don't see a good alternative solution to address this problem, so let's go with the best one we see now.